inference-engine/tests/functional/plugin/myriad/subgraph_tests/trivial_graphs.cpp

```diff
@@ -0,0 +1,88 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <shared_test_classes/base/layer_test_utils.hpp>
+
+#include <ngraph/opsets/opset6.hpp>
+#include <ie_ngraph_utils.hpp>
+
+#include "vpu/private_plugin_config.hpp"
+
+namespace {
+
+class ParameterResultTest : public testing::WithParamInterface<std::tuple<ngraph::Shape, ngraph::element::Type, std::string>>,
+                            virtual public LayerTestsUtils::LayerTestsCommon {
+protected:
+    void SetUp() override {
+        const auto& parameters = GetParam();
+        const auto& inShape = std::get<0>(parameters);
+        const auto& inType = std::get<1>(parameters);
+        targetDevice = std::get<2>(parameters);
```

Comment:
`std::tie` seems to be more safety

Reply:
I'm not sure it's more safety, but fixed anyway.